src/arrayMethodSort.js

```diff
@@ -5,7 +5,29 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    function swapItems(index) {
+      const temp = this[index];
+
+      this[index] = this[index - 1];
+      this[index - 1] = temp;
+    }
+
+    for (let i = 1; i < this.length; i++) {
+      if (compareFunction) {
```

Comment:
so to avoid this comparing create a `compareByDefault ` function that will be used in case we haven't passed an argument